lib/public/views/Runs/ActiveColumns/runsActiveColumns.js

```diff
@@ -78,9 +79,20 @@ export const runsActiveColumns = {
         visible: true,
         profiles: [profiles.none, 'lhcFill'],
         size: 'w-15 f6',
-        format: (detectors, run) => {
+        format: (_, run) => {
+            const detectors = run.detectorsQualities;
             if (detectors && detectors.length > 0) {
-                return [h('.badge.bg-gray-light.mh2.nDetectors-badge', run.nDetectors), `${detectors.toString()}`];
+                return [
```

Comment:
You should look at extracting this formatting in an individual component so that it can be reused.
For example, we already provide a similar component `formatRunDetectors.js` which does a similar job for `run-details` page. 
Add a new component, named something like `formatRunDetectorsInline.js` (to suggest this will be a list of detectors inline)